js/pages/settings.js

```diff
@@ -628,23 +634,23 @@ const updateUIAfterUnlink = async (isSuccess, type, error) => {
     
     if (isSuccess) {
       await refreshUserDataAfterEdit();
-        
-      const firebaseAuthEmail = userData[cId.firebaseAuthEmail] && !userData[cId.firebaseAuthEmail].startsWith('noreply') ? userData[cId.firebaseAuthEmail] : ''; 
-      const firebaseAuthPhone = formatFirebaseAuthPhoneNumber(userData[cId.firebaseAuthPhone]);
 
-      if (firebaseAuthEmail && type !== 'email') {
+      optVars.loginEmail = userData[cId.firebaseAuthEmail] && !userData[cId.firebaseAuthEmail].startsWith('noreply') ? userData[cId.firebaseAuthEmail] : ''; 
+      optVars.loginPhone = formatFirebaseAuthPhoneNumber(userData[cId.firebaseAuthPhone]);
```

Comment:
Optional: Check for `firebaseAuthPhone` exists before passing it to function

Reply:
Leaving this as is only because the function handles it and it is well-tested. I see the value in this suggestion 👍